app/src/main/java/com/framgia/forder/data/source/remote/api/request/ShopRequest.java

```diff
@@ -0,0 +1,141 @@
+package com.framgia.forder.data.source.remote.api.request;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+import com.framgia.forder.data.model.CollectionAvatar;
+import com.framgia.forder.data.model.CollectionImage;
+import com.framgia.forder.data.model.Image;
+import com.framgia.forder.data.model.Shop;
+import com.framgia.forder.data.model.User;
```

Comment:
ERROR: Unused import: com.framgia.forder.data.model.User. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports